### DISCUSSION TITLE
Waste and Waste Treatment in Processes

### DISCUSSION THREAD

**Discussion by yviph** [LCA-ActivityBrowser/activity-browser#974] [Q&A] [answered]
Hi all,

I am quite new to AB, but need to modell a couple of own processes...

A few processes produce different kind of waste as an output, which need treatment and I am wondering if I need to put the amount of waste and the process of waste treatment into the Technosphere Flows? Or is it sufficient to add the waste treatment process and define the amount that needs to be treated?

I was looking for an answer online and on github, but haven't found anything...

Thanks and kind regards,
Yvonne  :)

**Comment by marc-vdm** [comment, marked as answer]
Activity Browser and Brightway are closely linked to how ecoinvent models data, because of that, it is probably easiest to model your data in the same way. Ecoinvent does not model 'waste' (a physical flow) but instead models 'waste treatment' (a service) with negative numbers. The waste is listed in the 'technosphere flows' because it's not the functional flow of the process, but we use the negative number to show that it still 'leaves' the process instead of being an input.

That would look something like this:

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/34626062/1eef3ab5-fa34-4db5-ad15-9396b3f7bba7)

![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/34626062/9afa86e0-ea6c-4805-b33a-e6b9d4dd8f46)
![image](https://github.com/LCA-ActivityBrowser/activity-browser/assets/34626062/3e1cb97b-78d9-4ca6-9dcd-07b28e3a5316)

Here's also the example file I made, you can import that into AB to see what it look like if that helps!
[waste example.xlsx](https://github.com/LCA-ActivityBrowser/activity-browser/files/11572881/waste.example.xlsx)
